src/base/async/int.c

```diff
@@ -2007,9 +1986,13 @@ static int redir_it(void)
     LWORD(ebx) = DOS_SUBHELPER_MFS_REDIR_INIT;
     LWORD(eax) = DOS_HELPER_MFS_HELPER;
     if (mfs_inte6() == TRUE && LWORD(eax)) {
-	redirect_devices();	/* We have a functioning redirector so use it */
+        /* We have a functioning redirector so use it */
+        LWORD(eax) = 0x5f01; // set redirection mode
```

Comment:
Where does this come from?
Can't find in RBIL.

Reply:
http://www.ctyme.com/intr/rb-3041.htm

Reply:
Hmm, is it really the global call,
for all drives and printers? Then I
find it good, but can't it be used
still from emufs.sys?

Reply:
Yep it's a global call that ends up in the redirector code. That allows us to use all those int2f/12 functions that need to be called on the DOS stack.
Mmm it breaks the limited DOS function rule for device driver init routines, there's no guarantee that DOS is ready to make redirector calls. I think there is more than just one reason that CDROM access was split into devicedriver and TSR redirector parts.

Reply:
But what exactly 2f/12 funcs do we need?

Reply:
int2f/1217 get CDS for specific drive, maybe others